packages/counter/src/styles/Counter.module.css

```diff
@@ -1,8 +1,9 @@
+/* stylelint-disable number-max-precision */
 .Counter {
   display: inline-block;
   font-size: 0.6875rem;
   font-weight: var(--lp-font-weight-bold);
-  line-height: var(--lp-line-height-200);
+  line-height: 1.285714286;
```

Comment:
why do we have this very specific number here?

Reply:
The old value for `lp-line-height-200`